starcheck/pcad_att_check.py

```diff
@@ -0,0 +1,44 @@
+import hopper
+
+def make_pcad_attitude_check_report(backstop_file, or_list_file=None,
+                                    ofls_characteristics_file=None, out=None,
+                                    ):
+    """
+    Make a report for checking PCAD attitudes
+
+    """
+
+    # Run the commands and populate attributes in `sc`, the spacecraft state.
+    # In particular sc.checks is a dict of checks by obsid.
+    # Any state value (e.g. obsid or q_att) has a corresponding plural that
+    # gives the history of updates as a dict with a `value` and `date` key.
+    # This does not define the initial state from continuity at this time.
+    sc = hopper.run_cmds(backstop_file, or_list_file, ofls_characteristics_file,
+                         initial_state=None)
```

Comment:
I just don't think we've parsed the attitude history in starcheck up to
this point and didn't know if we needed it just to run hopper for this.